test/release.test.js

```diff
@@ -570,3 +572,43 @@ tap.test(
     sinon.assert.notCalled(stubs.notifyIssuesStub)
   }
 )
+
+tap.test('Should fail when release is not found', async () => {
```

Comment:
don't we need tests also for the PR part of this action, to cover the error cases you added in this PR?

Reply:
Yep, added a new test for that case